### DISCUSSION TITLE
These new badges are too cartoonish

### DISCUSSION THREAD

**Discussion by GoldenMaximo** [community/community#27413] [Other Feature Feedback, Questions, & Ideas]
They look like they're made for children.

I kinda dig the idea of having achievements like StackOverflow, LeetCode and CodeWars, but when they look like infantile corporate memphis it completely ruins it for me.

You know those gen z developers who love talking about their Pokemon apps and Medium articles? To me, they're exactly the target audience for these badges.

I really want to enjoy these badges, I do! I like the idea (kind of). But I don't live in San Fracisco, I don't own a Prius and I don't consume soy products.

**Comment by radar-blip** [comment]
Right. They should have python, and C++ badges, and even shell badges.

**Reply by jjaaster** [reply]
I agree, different language badges to earn towards proving one's ability to show that coding ability & accomplishment.

**Comment by Hoijof** [comment]
So, probably adding an option to disable the badges on your profile would be good feedback too.

**Reply by baysideengineer** [reply]
Hey @Hoijof, you can go to **settings -> public profile -> profile settings -> show achievements on my profile** and turn them off.

**Reply by Hoijof** [reply]
Excellent! Thanks! :D

**Comment by jfullstackdev** [comment]
yes, you have the option to disable it, even your profile to be as private, you were given the options, because not all users have the same perception of these badges, we need to consider them too

**Comment by Pressacco** [comment]
Agreed - I like the badges idea... but they could look more professional.

**Comment by St3fM4ury** [comment]
@GoldenMaximo
I'm not young and didn't eat tofu but I like this kind of design for badges (round and modern). I can't imagine a simple square box with C# wrote in the middle...

**Comment by Rhys-Alexander** [comment]
It fits the Github design, beyond that does it really matter that much?

**Comment by bryan-stapleton** [comment]
I came here to give this feedback as well. 

Additionally, the "YOLO" badge seems wholly negative. I don't want that on my page just because I approved my own personal projects PR that I wrote without reviewing it. I wrote it and I'm the only contributor. There was nothing yolo about it.

Why would anyone even want to give the appearance of yoloing something as complex as a coding project? Some of these just don't seem well thought out, no offense. "PR Shark" or "YOLO" doesn't tell me anything useful. I like the concept but just having language badges i.e. "typescript", "python" and something like "regular contributor" or "open source contributor" would be nice. Thanks.

**Reply by GoldenMaximo** [reply]
I Agree 200%

**Reply by jfullstackdev** [reply]
as from the other discussions, maybe what we can request is to have an option to select only the badges to be displayed because as of now, it's all or nothing

PR Shark means you have so many pull requests merged, so it makes sense, you're an active contributor

![image](https://user-images.githubusercontent.com/47092464/181803101-9a94e224-04fe-4e56-8ca1-2ccd045d87f7.png)

I have up to 128th pull request merged

as for the YOLO badge, this is, in my opinion, in the context when collaborating, closing it without a review seems all agreed and confident, hence the badge

**Reply by rowan-mcalpin** [reply]
The YOLO badge seems like a horrible idea, since it will encourage users to approve a PR without a review, which isn't a good thing in the context of group projects.

**Comment by BorislavChernev** [comment]
You can hide your achievements by going to: your profile -> settings -> public profile -> scroll down to and you will see: "Show Achievements on my profile".

**Comment by jfullstackdev** [comment]
latest update, you can now pick what you want to hide, unlike in the past, it was all or nothing

go to Achievements Tab, click the `Achievements` word, it is a link

![image](https://user-images.githubusercontent.com/47092464/204711684-e7c5ebb6-57f9-4fbf-b8fe-96c2182f41bb.png)

or this URL, it's same thing 

https://github.com/[username]?tab=achievements

you should be seeing like this 

![image](https://user-images.githubusercontent.com/47092464/204711801-dde96c4d-5133-4201-ad8a-9b23b55ed5d2.png)

then, click any badge, hover left then you should be seeing like this 

![image](https://user-images.githubusercontent.com/47092464/204711940-2c5e8b4d-7580-4145-afc0-268f1739d8de.png)

then boom ... you now have the option to hide those badges that you want to hide
